### DISCUSSION TITLE
Audi A3 MIB HIGH Harman, stuck at 11% initializing navigation

### DISCUSSION THREAD

**Discussion by Vengo222** [Mr-MIBonk/M.I.B._More-Incredible-Bash#312] [General]
Hi

I have an Audi A3 to which I updated the firmware to the firmware version MHI2_ER_AU37x_P5089 before patching with M.I.B, after a cold start it hangs at 11% initializing the navigation, it does not restart, but it does not respond to any key, it does not restart, it just remains locked at 11%

After installing M.I.B to patch the unit and update maps, in the fecs list the line 023000EE appears in an illegal state, to try to solve the problem, I restored feccontainer from M.I.B, but this did not solve the illegal state of the line either, so I uninstalled M.I.B, to do a clean installation the next day, but in the morning the navigation does not start, the screen stays at 11% of the navigation start, and no key works, nothing works... How can I solve the problem?, I made a backup on the SD Card.

attached log
[MU1326-JTH-07620.07.17340503CA.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/12339643/MU1326-JTH-07620.07.17340503CA.log)

**Comment by harman-f** [comment]
Looking at the LOG file from your 1st post, you did not patch the unit, just run addFec.

You might have started a recovery, however the LOG is not complete here.
looks more like you might have powered down the unit....

`search JTH-07620.07.17340503CA Backup File MU1326-ifs-root-part2-0x00ba0000.ifs

[flashlock] unlocking flash: DONEFlashing on OFFSET 0x00ba0000...

There will be no further output until flash is finished! `

**Reply by Vengo222** [reply]
How can I recover the system?

Do you have a link to the information to perform that recovery?

Can it be recovered via D-Link using a script like the link information I wrote above?

**Reply by harman-f** [reply]
Check mibwiki.one there are some instruction. 
you need a D-Link to start with. 

If that can not connect anymore maybe UART. START with D-Link.

**Comment by Vengo222** [comment]
solved, thanks

**Reply by bwilliams75** [reply]
How did you solve this? I am having this issue now.

**Comment by Vengo222** [comment]
I have another query:

In this family we have another Audi TT model, it has 2 USB inputs but it is not equipped with navigation, it does not have the NAVI symbol on the screen, and it only has the wide screen in the copckit, and I think that with M.I.B it is possible to add the navigation option to it Please tell me what the procedure would be.

Of course, first the backup, then the patching, and after checking on the screen that the map and navigation FECs have been added, use the "navigation activation" function and them load the map file? Can you tell me what it would be in order, or if there is another element that I have to activate or add.   

Thank you

**Comment by Vengo222** [comment]


**Reply by harman-f** [reply]
Just run the AIO patch.

**Reply by Vengo222** [reply]
Ok, thanks

**Comment by Vengo222** [comment]
Hi

in the Audi TT, I have been able to activate the navigation function and load the 2023/2024 map, but to use the navigation function I can only do it through voice recognition, with the steering wheel key, the navigation icon does not appear on the screen, and I cannot access it either to navigation using the Map/Nav key on the keyboard.

Do you know of a way to make the navigation icon appear on the screen?, or that it can be accessed using the Nav/map key on the central keyboard?

The other problem that I have is that the map is not shown in panoramic view, there are only indicative arrows on the right side of the copkit, I suppose that the view mode will depend on some configuration to be carried out through ODIS, if you know something about this, I would appreciate the information.

**Comment by Vengo222** [comment]
Actually the solution for everything to work correctly would be:

  - Enable the Nav/map key on the central keyboard, which is currently inactive.

- Code the screen so that at the top you can select the navigation option, which would allow you to use the left/right buttons on the steering wheel, which would allow you to select the views and control the navigation options from the left side of the steering wheel.

- Code the screen so that the navigation and map option appears in the main menu.

I think that all this could be done through the VCDS program, with some long code, or with some coding in the Dashboard section, any information will be appreciated.

**Reply by Chrisnickname** [reply]
Hi @Vengo222 how did you succeed to fix the problem with navigation stuck 11% issue ?
I’ m sure it can help me 
Thanks

**Comment by Tomt97** [comment]
check out this discussion., https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/448   found a fix for mine

**Comment by Chrisnickname** [comment]
Hi,
Very helpfull
I have coded my car and now it’s no more stuck.
I have a message saying that maps are not usable
But I will try to import the old maps , keep in touch.

Chris

**Reply by PepeBarcelona** [reply]
Hi @Chrisnickname , how did you fix your 11% issue? Did you do what @fuadamin says in the post #448 ?
I have already checked bytes 0, 1, 2, and 3 in my the long coding of my 5F module, but everything is ok. So, I am kind of lost.

Thanks in advance.

**Comment by Chrisnickname** [comment]
Hi,
Thank you again for the advice !
After updating the entire Maps, everything is Working now !
I’m so happy 😉
2 years of wait 😂



Christophe

**Reply by Tomt97** [reply]
awesome.   good stuff

**Comment by Chrisnickname** [comment]
Hi,
Yes i have done the check and the setup as it was in the post.
Using mib toolbox and model_db.
That was my issue !
For my car the good value was 013305 (Audi TT).
Hope it will help you




Christophe Envoyé avec Gmail Mobile


Le sam. 14 sept. 2024 à 07:34, PepeBarcelona ***@***.***> a
écrit :

**Comment by Wooody1982** [comment]
Hi, what was the final solution? I have the same issue. It stock at 11 percent. Update maps e.g.`? Best

**Reply by PepeBarcelona** [reply]
Hi @Wooody1982, have a look at this article (it is in russian, but google translation is quite accurate)
https://www.drive2.ru/l/684029549335300827/

There you will find the step by step guide to solve the problem.
I followed the steps and it worked for me.
